lib/ADS/ADS.cpp

```diff
@@ -0,0 +1,72 @@
+#include "ADS.h"  
+
+ADS::ADS(){}
+ADS::ADS(Adafruit_ADS1115 _Right, Adafruit_ADS1115 _Left, Adafruit_ADS1115 _Back)
+    : right(_Right), left(_Left), back(_Back) {}
+
+
+bool ADS::initializeAdafruit(){
+    right.setGain(GAIN_FOUR);
+    left.setGain(GAIN_FOUR);
+    back.setGain(GAIN_FOUR);
+    if (!right.begin(0x48)||!left.begin(0x49)||!back.begin(0x4A)) {
+        return false;
+    }
+}
+double ADS::getAverageRight(){
+    int16_t ads5=right.readADC_SingleEnded(0);
+    int16_t ads6=right.readADC_SingleEnded(1);
+    int16_t ads7=right.readADC_SingleEnded(2);
+    int16_t ads8=right.readADC_SingleEnded(3);
```

Comment:
If these arent and wont be changed, can we set these variables as const 
```suggestion
    const int16_t ads5=right.readADC_SingleEnded(0);
    const int16_t ads6=right.readADC_SingleEnded(1);
    const int16_t ads7=right.readADC_SingleEnded(2);
    const int16_t ads8=right.readADC_SingleEnded(3);
```